tests/unit/core/reducers/test_categories.js

```diff
@@ -202,6 +231,22 @@ describe('categories reducer', () => {
           },
           [ADDON_TYPE_LANG]: {},
           [ADDON_TYPE_OPENSEARCH]: {},
+          [ADDON_TYPE_STATIC_THEME]: {
+            anime: {
+              ...fakeCategory,
+              application: CLIENT_APP_FIREFOX,
+              name: 'Anime',
+              slug: 'anime',
+              type: ADDON_TYPE_STATIC_THEME,
+            },
+            naturé: {
+              ...fakeCategory,
+              application: CLIENT_APP_FIREFOX,
+              name: 'Naturé',
+              slug: 'naturé',
+              type: ADDON_TYPE_STATIC_THEME,
+            },
+          },
```

Comment:
Why do you add these changes here? Because we have a new type of add-ons?

Reply:
Yeh I know it's doesn't seem exactly related to the changes I made here but it was failing here without these updates. Is this ok?

Reply:
I am not sure to understand why it is failing, which change requires these updates?

Reply:
I actually was able to remove some of the updates but its still looking for some static theme info it seems:
![Alt text](https://monosnap.com/image/B4UfgGttLFwoHbhF7rPCvPE31uh97x.png)